libraries/AP_BattMonitor/AP_BattMonitor_Params.h

```diff
@@ -18,7 +18,12 @@ class AP_BattMonitor_Params {
         BattMonitor_LowVoltageSource_SagCompensated = 1
     };
     enum class Options : uint8_t {
-        Ignore_UAVCAN_SoC = (1U<<0),
+        Ignore_UAVCAN_SoC                   = (1U<<0),
+        UAVCAN_MPPT_Use_Input_Value         = (1U<<1),
```

Comment:
We should add all these bits to the _OPTIONS parameter description

Reply:
Oh, whoops! That must have fallen off at some point. I'll add it back in.

Reply:
thanks for including them in your commit